Documentation/Contributing/development-flow.md

```diff
@@ -245,7 +245,7 @@ Common cases that may need tests:
 Rook's upstream continuous integration (CI) tests will run integration tests against your changes
 automatically.
 
-Integration tests will be run in Github actions. If an integration test fails, a tmate session will be available for troubleshooting for a short time. See the action details for an ssh connection to the Github runner.
+Integration tests will be run in Github actions. If an integration test fails, restart CI with debug mode from github UI or add label `debug-ci`  and push your changes again and a tmate session will be available for troubleshooting for a sometime. See the action details for an ssh connection to the Github runner.
```

Comment:
Can we restructure this with another header and bullets?
```suggestion

### TMate Session

Integration tests will be run in Github actions. If an integration test fails, enable a tmate session to troubleshoot the issue by one of the following steps:
- Restart the CI action and enable the "Debug" mode checkbox from the github UI, or
- Add the label `debug-ci` to the PR and push your changes again. 

See the action details for an ssh connection to the Github runner.
```

Reply:
done